src/Aspire.Hosting/ApplicationModel/ContainerMountAnnotation.cs

```diff
@@ -14,11 +14,11 @@ public sealed class ContainerMountAnnotation : IResourceAnnotation
     /// <summary>
     /// Instantiates a mount annotation that specifies the source and target paths for a mount.
     /// </summary>
-    /// <param name="source">The source path of the mount.</param>
+    /// <param name="source">The source path if a bind mount or name if a volume. Can be <c>null</c> if the mount is an anonymous volume.</param>
```

Comment:
Just FYI, DCP does not support anonymous volumes as of today (you have to give the volume a valid name). We could add that, but that the state as of today. Not sure if you want to hold off on the part of introducing explicit support for anonymous volumes, or do the whole change anyway and document the missing functionality.

Reply:
Umm, I just tried it and it worked?

Reply:
OK. If it works, let's proceed ☺️ 

TBH the code in DCP does not really handle the anonymous volume scenario, but I can also see how it happens to work anyway. 😆

Reply:
Yeah I think it's as simple as if you ask for a volume and the source isn't specified you get an anonymous volume 😃